fastlane/metadata/android/tr/full_description.txt

```diff
@@ -0,0 +1,38 @@
+Pek çok insan için, ufak bir arka plan gürültüsü sakinleşmeye ve odaklanmaya
+yardımcı oluyor. Bazı durumlarda üretkenliği bile arttrıyor. Ayrıca arka plan
+sesi (ambiyans) stresin etkilerini azaltmaya da yardımcı olabilir.
+
+Noice çevresel sesleri kullanarak kendi ambiyans kayıtlarınızı oluşturmanızı
+sağlayan bir uygulamadır. İşinize odaklanmak ya da sakin, barış dolu bir ortam yaratarak uyumanızı sağlamak adına rahatsız edecek sesleri baskılayacak mükemmel kayıtlarınızı oluşturun.
```

Comment:
New lines will start new paragraphs. Please remove them to keep paragraphs as single lines.

```suggestion
Pek çok insan için, ufak bir arka plan gürültüsü sakinleşmeye ve odaklanmaya yardımcı oluyor. Bazı durumlarda üretkenliği bile arttrıyor. Ayrıca arka plan sesi (ambiyans) stresin etkilerini azaltmaya da yardımcı olabilir.

Noice çevresel sesleri kullanarak kendi ambiyans kayıtlarınızı oluşturmanızı sağlayan bir uygulamadır. İşinize odaklanmak ya da sakin, barış dolu bir ortam yaratarak uyumanızı sağlamak adına rahatsız edecek sesleri baskılayacak mükemmel kayıtlarınızı oluşturun.
```